plugin/src/it/advanced/backend/src/main/java/com/spotify/it/backend/Main.java

```diff
@@ -1,22 +1,3 @@
-/*
- * -/-/-
- * Dockerfile Maven Plugin
- * %%
- * Copyright (C) 2015 - 2016 Spotify AB
- * %%
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- * 
- *      http://www.apache.org/licenses/LICENSE-2.0
- * 
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * -\-\-
- */
```

Comment:
I removed the old file headers with sed and then ran `mvn license:update-file-header`. I guess it didn't update this file because the directory structure is different, ie doesn't live inside a dir called `java`?